lib/nodejs/lib/thrift/connection.js

```diff
@@ -221,6 +221,10 @@ Connection.prototype.connection_gone = function () {
   });
 
   this.retry_timer = setTimeout(function () {
+    if (self.connection.destroyed) {
+      return;
```

Comment:
```suggestion
      self.retry_timer = null;
      return;
```

Based on the way the code is written below...

Reply:
Thanks for the suggest. Changed.